guides/installation/template.md

```diff
@@ -66,7 +66,13 @@ Add a new software repository to your system to have the latest PHP version.
 ```bash
 sudo add-apt-repository ppa:ondrej/php
 
+# for shopware >6.5.0.4 and <6.6.x
 sudo apt-get install -y php8.1-fpm php8.1-mysql php8.1-curl php8.1-gd php8.1-xml php8.1-zip php8.1-opcache php8.1-mbstring php8.1-intl php8.1-cli
+
+# for Shopware 6.6x
+sudo apt-get install -y php8.2-fpm php8.2-mysql php8.2-curl php8.2-gd php8.2-xml php8.2-zip php8.2-opcache php8.2-mbstring php8.2-intl php8.2-cli
```

Comment:
I would change it straight to 8.2 and delete the other ones. We have versioning per shopware version, so when you want to see the older version, just switch :)

Reply:
Fine by me, if you expect me to answer

Reply:
can you change that 👍

Reply:
done